src/MiniCover/Commands/BaseCommand.cs

```diff
@@ -51,32 +52,22 @@ private void ValidateOptions()
             }
         }
 
-        private void InitializeExecution()
-        {
-            OnExecute(() =>
-            {
-                ValidateOptions();
-                Line($"Starting command: {Description}");
-                return Execution();
-            });
-        }
-
-        protected void PositiveLine(string message)
+        protected void WritePositiveLine(string message)
         {
             Console.ForegroundColor = ConsoleColor.Green;
             Console.WriteLine(message);
             Console.ResetColor();
         }
 
-        protected void BadLine(string message, Exception exception = null)
+        protected void WriteNegativeLine(string message, Exception exception = null)
```

Comment:
It should be ILogger but in MinCover we don't have a di :(

Reply:
Yes, MiniCover started very simple and quickly written.
I created it to run coverage on Linux at my previous job.
I was sure MS would release their cross platform VSTest soon.
Thanks to efforts like yours it is growing more than I was planning to 😄

Reply:
I mean, it's getting new features and nice refactors. That's good.